pkg/controller/podautoscaler/horizontal.go

```
@@ -315,7 +327,8 @@ func (a *HorizontalController) computeReplicasForMetrics(ctx context.Context, hp
 		return 0, "", statuses, time.Time{}, fmt.Errorf("invalid metrics (%v invalid out of %v), first error is: %v", invalidMetricsCount, len(metricSpecs), invalidMetricError)
 	}
 	setCondition(hpa, autoscalingv2.ScalingActive, v1.ConditionTrue, "ValidMetricFound", "the HPA was able to successfully calculate a replica count from %s", metric)
-	return replicas, metric, statuses, timestamp, nil
+
+	return replicas, metric, statuses, timestamp, fmt.Errorf("invalid metrics (%v invalid out of %v), first error is: %v", invalidMetricsCount, len(metricSpecs), invalidMetricError)
```

We used to silence errors on scale-ups? Good that we're fixing this 👍